ui/scripts/network.js

```diff
@@ -412,8 +412,6 @@
                         $.ajax({
                             url: createURL('listRemoteAccessVpns'),
                             data: {
-                                account: g_account,
```

Comment:
Is this call only used in the project view?
I mean, if I list networks and I am out of the project view, should not ACS filter by these data? Or, is it done automatically in the server side when the users is logged-in?